crates/biome_js_formatter/src/ts/objects/literal_enum_member_name.rs

```diff
@@ -0,0 +1,10 @@
+use crate::prelude::*;
+use biome_js_syntax::TsLiteralEnumMemberName;
+use biome_rowan::AstNode;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatTsLiteralEnumMemberName;
+impl FormatNodeRule<TsLiteralEnumMemberName> for FormatTsLiteralEnumMemberName {
+    fn fmt_fields(&self, node: &TsLiteralEnumMemberName, f: &mut JsFormatter) -> FormatResult<()> {
+        format_verbatim_node(node.syntax()).fmt(f)
```

Comment:
This is incorrect @Conaclos, it's still using the verbatim formatting

Reply:
Thanks!